### ISSUE TITLE
Version 1.6.2 gets flagged as Trojan

### ISSUE THREAD

**Issue by Thunderhawk2001** [meetrevision/revision-tool#63]
https://www.virustotal.com/gui/file/0c1b82c60d06f0dc41afbda61bf2282f6d2fe0e3e3d99a17c18d0d8998abeb09

Newest version get's flagged as a trojan by 50% of VT scan engines. I think that's a false positive?

**Comment by melo936** [comment]
Yes, it is a false positive. Revision Tool is built by GitHub Actions, thus as far as I'm aware it isn't possible to modify the process. Moreover, the tool isn't digitally signed which is expected to happen.

**Comment by notguoxin** [comment]
I can confirm, it was flagged by Windows Defender.
![image](https://github.com/meetrevision/revision-tool/assets/121658107/bad893f9-87fa-4507-8309-7b7ca7d19240)

**Comment by Pinghigh** [comment]
It flagged as Trojan as well in Kaspersky Free 
![image](https://github.com/meetrevision/revision-tool/assets/77083712/ca610777-d78d-474c-a018-51538fcbca2b)

**Comment by lhs5e** [comment]
![image](https://github.com/meetrevision/revision-tool/assets/74890071/0d48ba40-f793-4667-bb62-a0588677cb3f)

Most detections are due to script to disable Windows Defender

**Comment by Pinghigh** [comment]
> ![image](https://private-user-images.githubusercontent.com/74890071/295216511-0d48ba40-f793-4667-bb62-a0588677cb3f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3MzM3NDEsIm5iZiI6MTcxNDczMzQ0MSwicGF0aCI6Ii83NDg5MDA3MS8yOTUyMTY1MTEtMGQ0OGJhNDAtZjc5My00NjY3LWJiNjItYTA1ODg2NzdjYjNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTAzVDEwNTA0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNmOWNmYTg1ZDJjMzAwYjAxMDU5ODYyZjAyZThiNDg0N2VlNmVlNjU2YmZmZDY4MjVhZmM2ZTdlNjJhZDkzZTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Y-CGOSv2leCnzYBwrzjmvHArSUQwvx-wu-S1CAUBnXA)
> 
> Most detections are due to script to disable Windows Defender

Kaspersky's detection as well
